app/join.tsx

```
@@ -1,88 +1,61 @@
-                        } />
-
-
-                </View>
-
-
-                <View className="my-5">
+                <Text className="text-2xl font-bold dark:text-white text-center mb-2">
+                    Rejoindre un voyage
+                </Text>
+                <Text className="text-md text-gray-500 dark:text-gray-400 text-center mb-8">
+                    Collez le lien pour rejoindre l'aventure
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)